tests/mdev_configuration_allocation_test.go

```diff
@@ -354,6 +360,174 @@ var _ = Describe("[Serial][sig-compute]MediatedDevices", Serial, decorators.VGPU
 			Eventually(checkAllMDEVCreated(newDesiredMdevTypeName, newExpectedInstancesNum), 3*time.Minute, 15*time.Second).Should(BeInPhase(k8sv1.PodSucceeded))
 
 		})
+		It("should successfully create VMI when guest memory is less than or equal to request memory", func() {
+			By("Creating a Fedora VMI")
+			networkData := cloudinit.CreateDefaultCloudInitNetworkData()
+			vGPU := v1.GPU{
+				Name:       "gpu1",
+				DeviceName: deviceName,
+			}
+
+			vmi = libvmi.New(
+				libvmi.WithRng(),
+				libvmi.WithContainerDisk("disk0", cd.ContainerDiskFor(cd.ContainerDiskFedoraTestTooling)),
+				libvmi.WithInterface(libvmi.InterfaceDeviceWithMasqueradeBinding()),
+				libvmi.WithNetwork(v1.DefaultPodNetwork()),
+				libvmi.WithCloudInitNoCloudNetworkData(networkData),
+				libvmi.WithGuestMemory("512Mi"),
+				withGPUDevice(vGPU),
+			)
+
+			createdVmi, err := virtClient.VirtualMachineInstance(testsuite.GetTestNamespace(vmi)).Create(context.Background(), vmi)
+			Expect(err).ToNot(HaveOccurred())
+			vmi = createdVmi
+			libwait.WaitUntilVMIReady(vmi, console.LoginToFedora)
+		})
+		It("should ignore memory-overcommit if the resulting request memory is less than guest memory", func() {
+			By("Applying cluster memory-overcommit")
+			config.DeveloperConfiguration.MemoryOvercommit = 200
+			tests.UpdateKubeVirtConfigValueAndWait(config)
+
+			By("Creating a Fedora VMI")
+			networkData := cloudinit.CreateDefaultCloudInitNetworkData()
+			vGPU := v1.GPU{
+				Name:       "gpu1",
+				DeviceName: deviceName,
+			}
+
+			vmi = libvmi.New(
+				libvmi.WithRng(),
+				libvmi.WithContainerDisk("disk0", cd.ContainerDiskFor(cd.ContainerDiskFedoraTestTooling)),
+				libvmi.WithInterface(libvmi.InterfaceDeviceWithMasqueradeBinding()),
+				libvmi.WithNetwork(v1.DefaultPodNetwork()),
+				libvmi.WithCloudInitNoCloudNetworkData(networkData),
+				libvmi.WithGuestMemory("512Mi"),
+				withGPUDevice(vGPU),
+			)
+
+			createdVmi, err := virtClient.VirtualMachineInstance(testsuite.GetTestNamespace(vmi)).Create(context.Background(), vmi)
+			Expect(err).ToNot(HaveOccurred())
+			vmi = createdVmi
+			libwait.WaitUntilVMIReady(vmi, console.LoginToFedora)
+			Expect(vmi.Spec.Domain.Resources.Requests.Memory().String()).To(Equal("4096M"))
+		})
+		It("consume hugepages when page size is greater than request memory", func() {
+			var hugepagesVmi *v1.VirtualMachineInstance
+
+			verifyHugepagesConsumption := func() bool {
+				// TODO: we need to check hugepages state via node allocated resources, but currently it has the issue
+				// https://github.com/kubernetes/kubernetes/issues/64691
```

Comment:
This TODO seems outdated, the issue was closed with some explanations of what to do instead.
Might be worth creating a (good first) issue to investigate.